deployments/k8s/khaas/values.yaml

```diff
@@ -0,0 +1,55 @@
+team: <your_team>
+services:
+  ingestor:
+    name: ghcr.io/datadog/kubehound-ingestor
+    version: latest
```

Comment:
The fact that it's called `name` and not `image` or `image` and a sub field `name` is confusing.
It feels like its the name of the pod or the name of the service